### PR TITLE
Ehance apache test case and fix database table fields

### PR DESCRIPTION
**Pull request by lubaihua33**
1. Added "-s timeout" for ab command.
2. Fix some fields of Apache and Memcached database table.

### CODE REVIEW

**Comment by lubaihua33** [comment]
```
[LISAv2 Test Results Summary]
Test Run On           : 08/11/2020 13:13:44
ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
Override VM size Under Test  : Standard_F72s_v2
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:10

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 BENCHMARK            PERF-APACHE-BENCHMARK-Synthetic                                                   PASS                 7.64 
      ARMImageName: Canonical UbuntuServer 18.04-LTS latest, Networking: Synthetic, OverrideVMSize: Standard_F72s_v2, TestLocation: westus2, Kernel Version: 5.3.0-1034-azure
      Concurrency=1 : requestsPerSec=337 meanConnectionTime_ms=3
      Concurrency=2 : requestsPerSec=802.21 meanConnectionTime_ms=2
```

**Comment by lubaihua33** [comment]
```
pload test results to database.
08/11/2020 13:24:02 : [INFO ] SQLQuery:  INSERT INTO Overlake_VM_Perf_Apache (TestCaseName,TestConcurrency,HostType,MeanConnectionTimes_ms,KernelVersion,WebServerVersion,Compl
eteRequests,InstanceSize,ConcurrencyPerAbInstance,TestDate,RequestsPerSec,HostOS,Document_bytes,NumberOfAbInstances,TransferRate_KBps,GuestOS,DataPath,HostBy) VALUES ('apache_
test','1','Azure','3','5.3.0-1034-azure','Apache/2.4.29','50000','Standard_F72s_v2','1','2020-08-11','337','18362-10.0-3-0.3034','204800','1','67475.1','ubuntu 18.04','Synthet
ic','westus2')
08/11/2020 13:24:02 : [INFO ] SQLQuery:  INSERT INTO Overlake_VM_Perf_Apache (TestCaseName,TestConcurrency,HostType,MeanConnectionTimes_ms,KernelVersion,WebServerVersion,Compl
eteRequests,InstanceSize,ConcurrencyPerAbInstance,TestDate,RequestsPerSec,HostOS,Document_bytes,NumberOfAbInstances,TransferRate_KBps,GuestOS,DataPath,HostBy) VALUES ('apache_
test','2','Azure','2','5.3.0-1034-azure','Apache/2.4.29','50000','Standard_F72s_v2','2','2020-08-11','802.21','18362-10.0-3-0.3034','204800','1','160621','ubuntu 18.04','Synth
etic','westus2')
08/11/2020 13:24:02 : [INFO ] Succeed to upload test results to database
Upload test results to database.
08/11/2020 13:25:38 : [INFO ] SQLQuery:  INSERT INTO Overlake_VM_Perf_Memcached (TestCaseName,AverageLatency_ms,WorstOpsPerSec,ConnectionsPerThread,HostType,BestOpsPerSec,AverageOpsPerSec,Kern
elVersion,Threads,WorstLatency_ms,InstanceSize,TestDate,HostOS,TestConnections,RequestsPerThread,GuestOS,DataPath,HostBy,BestLatency_ms) VALUES ('data_test','0.23833','3812.90','1','Azure','45
01.99','4178.72','5.3.0-1034-azure','1','0.26000','Standard_DS15_v2','2020-08-11','18362-10.0-2-0.2034','1','1000000','ubuntu 18.04','Synthetic','westus2','0.22000')
08/11/2020 13:25:38 : [INFO ] Succeed to upload test results to database
[LISAv2 Test Results Summary]
Test Run On           : 08/11/2020 13:06:29
ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:19

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 BENCHMARK            PERF-MEMCACHED-BENCHMARK-Synthetic                                                PASS                17.08 
      ARMImageName: Canonical UbuntuServer 18.04-LTS latest, Networking: Synthetic, TestLocation: westus2, Kernel Version: 5.3.0-1034-azure
      TestConnections=1 : AverageLatency_ms=0.23833 AverageOpsPerSec=4178.72
```

**Review by harz566 — APPROVED** [review]

**Review by LiliDeng — APPROVED** [review]

File: Testscripts/Windows/PERF-TWOVM-MIDDLEWARE-BENCHMARK.ps1 [hunk 1]
```diff
@@ -214,12 +214,9 @@ collect_VM_properties
                 $resultMap["HostType"] = $TestPlatform
                 $resultMap["HostBy"] = $CurrentTestData.SetupConfig.TestLocation
                 $resultMap["HostOS"] = $HostOs
-                $resultMap["GuestOSType"] = "Linux"
-                $resultMap["GuestDistro"] = $GuestDistro
-                $resultMap["GuestSize"] = $clientVMData.InstanceSize
+                $resultMap["GuestOS"] = $GuestDistro
+                $resultMap["InstanceSize"] = $clientVMData.InstanceSize
```
Comment (lizzha) [inline Testscripts/Windows/PERF-TWOVM-MIDDLEWARE-BENCHMARK.ps1:218]: did we change the database schema?

File: Testscripts/Linux/perf_apache.sh [hunk 2]
```diff
@@ -176,14 +177,14 @@ Run_ab()
 		concurrency_left=$(($concurrency_left - $concurrency_per_ab))
 		requests_left=$(($requests_left - $total_request_per_ab))
 		LogMsg "Running parallel ab command for: ${total_request_per_ab} X ${concurrency_per_ab}"
-		ab -n ${total_request_per_ab} -r -c ${concurrency_per_ab} http://${server}/test.dat >> ${log_folder}/${current_concurrency}.${testName}.bench.log & pid=$!
+		ab -n ${total_request_per_ab} -r -c ${concurrency_per_ab} -s $timeout http://${server}/test.dat >> ${log_folder}/${current_concurrency}.${testName}.bench.log & pid=$!
```
Comment (lizzha) [inline Testscripts/Linux/perf_apache.sh:180]: what's the default timeout, if not specifed?

**Review by lizzha — APPROVED** [review]

File: Testscripts/Linux/perf_apache.sh (hunk 2 above)
Comment (lubaihua33) [reply Testscripts/Linux/perf_apache.sh:180]: Default is 30s.

**Pull request merged**